### ISSUE TITLE
Feature Request: add application/sla mime type

### ISSUE THREAD

**Issue by hannemann** [MLAB-project/stl-thumbnailer#6]
After following your instructions only a few stl files have been thumbnailed.

I edited /usr/share/thumbnailers/stl.thumbnailer to look like this
```bash
[Thumbnailer Entry]
Exec=/usr/local/bin/stl_thumb.py %i %o %s
MimeType=application/sla;application/stl;model/x.stl-binary;model/x.stl-ascii
```
Suddenly all my minis appeared in nemo :)

**Comment by kaklik** [comment]
It is interesting that works for you. Because we have troubles with this bug: https://github.com/MLAB-project/stl-thumbnailer/issues/5

Have you working situation at Ubuntu 16.04 or 18.04 ?

**Comment by hannemann** [comment]
My machines run 18.04

**Comment by jamesthek** [comment]
Implemented in PR #8 .
